dipy/segment/tests/test_feature.py

```diff
@@ -284,6 +285,26 @@ def extract(self, streamline):
     d2 = metric.dist(features1, features2)
     assert_equal(d1, d2)
 
+    class ArcLengthFeature(dipymetric.Feature):
+        def infer_shape(self, streamline):
+            if sys.version_info > (3,):
```

Comment:
As @matthew-brett said I'm trying to test the Windows 64 bits behavior, i.e. where it uses `long` instead of `int` for constant integers. I added a comment explaining  why we do such test.